pages/reference/OS/extended-support-release.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Extended Support Release Process
+excerpt: An overview of the {{ $names.company.lower }} extended support release process
+---
+# Extended Support Release (ESR) process
+
+__Note:__ This feature is only available on [Production and Enterprise plans][billing].
+
+The Extended Support Release (ESR) process allows fleet owners to update the host OS on their devices at most twice a year to ensure they are on a supported version. New ESR versions are released on a three-month schedule, and as such, fleet owners know in advance when releases will be available and when they need to upgrade.
+
+Each ESR release is supported for nine months from the time of release. A supported ESR version implies that high-risk security vulnerabilities and critical bug fixes will be backported to that version. Backports of any functional enhancements are not in scope.
+
+## ESR schedule
+
+Three lines of ESR versions are supported, known as _next_, _current_, and _sunset_. Every three months, a new ESR version is released, and each line will transition forward at that time as illustrated on the following diagram:
+
+![ESR schedule](/img/common/esr-process.png)
+
+For example, an ESR version is released known as `2019.10`, which becomes the _next-ESR_. In three months, a new ESR version, i.e., `2020.04`, becomes the _next-ESR_ line, while `2019.10` transitions to become the _current-ESR_. Similarly, three months after that, there is a new ESR version, and `2019.10` becomes the _sunset-ESR_. Three months later and nine months after release, the `2019.10` ESR version will reach its end-of-life, and no further updates will be offered, and users should update to one of the supported releases to receive future updates by performing a [self-service update][self-service-updates].
+
+__Note:__ ESR versions are named by year and month `yyyy.mm` and contain a patch version number. So, `2019.10.4` would be the fifth release of the `2019.10` ESR version.
```

Comment:
I wonder if it's worth clarifying this by adding something like " (`2019.10.0` would be the first release)" at the end? :man_shrugging: